tests/markdown_mixed_post.md

```diff
@@ -0,0 +1,7 @@
+```
+File with missing codeblock
+```
+
+```
+with no closing bracket
```

Comment:
Consider renaming the test files following the pattern `foo.md` and `foo_want.md`. That way, we're consistent between filenames and the tests themselves.

Reply:
Done.